### PR TITLE
Fix/post path

### PR DESCRIPTION
**Pull request by gbudau**
Co-authored-by: Victor Dedios <victor.dedios@outlook.com>

Fix POST upload with non-existing location path

### CODE REVIEW

File: srcs/incs/HttpPostResponse.hpp [hunk 1]
```diff
@@ -18,6 +18,7 @@
 #include <RequestConfig.hpp>
 #include <HttpBaseResponse.hpp>
 #include <HttpErrorResponse.hpp>
+#include <Utils.hpp>
```
Comment (Damrod) [inline srcs/incs/HttpPostResponse.hpp:21]: We have a few functions we don't use anymore in Utils...

File: srcs/incs/HttpPostResponse.hpp [hunk 2]
```diff
@@ -33,6 +34,7 @@ class HttpPostResponse: public HttpBaseResponse {
 		void	Upload_(File file);
 		void	HandleCGI_(File file);
 		void	HandleUpload_(File file);
```
Comment (Damrod) [inline srcs/incs/HttpPostResponse.hpp:36]: Is it necessary in all of these to pass the File by copy?

**Review by Damrod — APPROVED** [review]
LGTM!

File: srcs/app/http/response/File.cpp [hunk 3]
```diff
@@ -46,7 +46,8 @@ bool	File::HasEndSlash() const {
 	return file_path_[file_path_.size() - 1] == '/';
 }
 
-void	File::Upload(std::string upload_path, std::string content) {
+void	File::Upload(std::string filename, std::string content) {
```
[the remark this answers is not in the document]
Comment (gbudau) [reply srcs/app/http/response/File.cpp]: Good catch. Done.

File: srcs/incs/HttpPostResponse.hpp (hunk 1 above)
Comment (gbudau) [reply srcs/incs/HttpPostResponse.hpp:21]: :+1: I've removed them.

File: srcs/incs/HttpPostResponse.hpp (hunk 2 above)
Comment (gbudau) [reply srcs/incs/HttpPostResponse.hpp]: Not really, I've changed it.

**Comment by gbudau** [comment]
Thank you! :raised_hands:

**Pull request merged**